src/textual/widgets/_progress_bar.py

```diff
@@ -2,22 +2,48 @@
 
 from __future__ import annotations
 
-from math import ceil
+from math import ceil, cos, radians
 from time import monotonic
 from typing import Callable, Optional
+from typing_extensions import Literal
 
 from rich.style import Style
 
 from textual.geometry import clamp
 
 from ..app import ComposeResult, RenderResult
+from ..color import Color
 from ..containers import Horizontal
+from ..css._error_tools import friendly_list
 from ..reactive import reactive
 from ..renderables.bar import Bar as BarRenderable
 from ..timer import Timer
 from ..widget import Widget
 from ..widgets import Label
 
+BarThickness = Literal[0, 1, 2]
+"""The values of the valid bar thicknesses.
+
+These are the thicknesses that can be used with a [`Bar`][textual.widgets.Bar].
+"""
+_VALID_BAR_THICKNESSES = {0, 1, 2}
```

Comment:
These are ultimately used by the `BarRenderable` from `renderables.bar`, right?
So, I think these should be defined there and then imported.

Reply:
Ok